handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -30,9 +45,14 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 
 1. You [apply here](https://jobs.lever.co/sourcegraph/8b57b6f4-4eb7-4bcd-8143-8b147867522a/apply).
```

Comment:
@withdavidli can you give me the new apply link for the new requirement as we discussed so I can update it here?

Reply:
https://jobs.lever.co/sourcegraph/b2f9a8b0-cc06-4629-81a0-0f2fa64271c7/apply

created the new job req. will move applicants into new req.

Reply:
Thanks!